notifications_utils/logging.py

```diff
@@ -60,17 +60,16 @@ def after_request(response):
             })
```

Comment:
I'm sorry, changes here are not related to templates. The logic here was broken when `time_taken` was not defined, the call to the method `record_stats` was failing because the variable was not defined.

I took another look at the code and discovered that time_taken is expected to be in milliseconds. Values reported to StatsD were wrong.

https://github.com/jsocol/pystatsd/blob/f3f304b4b2c3d5eddeb9f4977d9c82c64c37a052/statsd/client/base.py#L26-L35